swagger/specification/ucp/resource-manager/UCP/preview/2023-10-01-preview/examples/AzureCredentials_List.json

```diff
@@ -0,0 +1,32 @@
+{
+  "operationId": "AzureCredentials_List",
+  "title": "List Azure credentials",
+  "parameters": {
+    "api-version": "2023-10-01-preview",
+    "planeType": "azure",
+    "planeName": "azurecloud"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
```

Comment:
Same question here (`values`)?

Reply:
It should be `value`: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections

All of our list APIs use `value` as the property name. If they don't then it's a bug.

Reply:
https://github.com/radius-project/radius/pull/7499/files#r1591556066